extensions/standard-processors/tests/unit/PutTCPTests.cpp

```diff
@@ -141,16 +120,16 @@ class PutTCPTestFixture {
   }
 
   size_t getNumberOfActiveSessions(std::optional<uint16_t> port = std::nullopt) {
-    if (auto session_aware_listener = dynamic_cast<ISessionAwareServer*>(getListener(port))) {
-      return session_aware_listener->getNumberOfSessions() - 1;  // There is always one inactive session waiting for a new connection
+    if (auto session_aware_listener = dynamic_cast<CancellableTcpServer*>(getListener(port))) {
+      return session_aware_listener->getNumberOfSessions();  // There is always one inactive session waiting for a new connection
```

Comment:
is the comment still relevant, why can we remove the `-1`?

Reply:
You are right, this comment makes no sense anymore (different method to count the sessions), I've removed the stale comment https://github.com/apache/nifi-minifi-cpp/pull/1457/commits/70d6a798090869c4c01b2dd7dc84eeaf9dcef2a3#diff-bbb56b112b398a6450c23e71e5dbbfb17bf20dad3d2fb908a1c6de26118f9ee8R124